projects/gis/src/app/main/main.component.ts

```diff
@@ -248,15 +246,4 @@ export class MainComponent implements OnInit, AfterViewInit {
     private windowHeight(): void {
         this.windowHeight$.next(window.innerHeight);
     }
-
-    private handleQueryParameters(): void {
-        const params = new URLSearchParams(window.location.search);
-        const sessionState = params.get('session_state');
-        const code = params.get('code');
-        const state = params.get('state');
-
-        if (sessionState && code && state) {
-            this.userService.oidcLogin({sessionState, code, state}).pipe(first()).subscribe();
-        }
-    }
```

Comment:
Why is this gone?

Reply:
this is now done in the user service. Previously, the query parameters were only ever checked AFTER the userService already tried to log in anonymously. In the case where the anonymous login was disabled this would lead to errors.